src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,104 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type AddMovie = {
```

Comment:
better to use interface for Props
```suggestion
interface Props = {
```